src/main/java/it/pagopa/pm/gateway/client/ContestoVO.java

```diff
@@ -0,0 +1,347 @@
+//
+// Questo file è stato generato dall'architettura JavaTM per XML Binding (JAXB) Reference Implementation, v2.3.0 
+// Vedere <a href="https://javaee.github.io/jaxb-v2/">https://javaee.github.io/jaxb-v2/</a> 
```

Comment:
What do you think about isolating the autogenerated classes in a package (for example generated.x.x) and not commit them? otherwise every local build would see some changes.

Reply:
@andrearovere for example https://github.com/pagopa/hub-pa-api/blob/a4824bd63b46a5db473520df2e5accf4b25d7457/reporting-batch/pom.xml#L211

Reply:
@andrearovere The idea is to generate the classes in target folder and then to merge them to classpath with an ad hoc maven plugin, the following configuration should help you
```
	<plugin>
	       <groupId>org.jvnet.jaxb2.maven2</groupId>
			<artifactId>maven-jaxb2-plugin</artifactId>
			<version>0.14.0</version>
			<executions>
				<execution>
					<goals>
						<goal>generate</goal>
					</goals>
				</execution>
			</executions>
			<configuration>
				<schemaLanguage>WSDL</schemaLanguage>
				<generateDirectory>${project.basedir}/target/generated</generateDirectory>
				<generatePackage>it.pagopa.pm.gateway.client</generatePackage>
				<schemaDirectory>${project.basedir}/src/main/resources/wsdl</schemaDirectory>
				<schemaIncludes>
					<include>PagoPaServiceFacade.wsdl</include>
				</schemaIncludes>
			</configuration>
		</plugin>
            <plugin>
                <groupId>org.codehaus.mojo</groupId>
                <artifactId>build-helper-maven-plugin</artifactId>
                <version>3.2.0</version>
                <executions>
                    <execution>
                        <id>add-source</id>
                        <phase>generate-sources</phase>
                        <goals>
                            <goal>add-source</goal>
                        </goals>
                        <configuration>
                            <sources>
                                <source>target/generated</source>
                            </sources>
                        </configuration>
                    </execution>
                </executions>
            </plugin>

```